components/collection/drop/utils.ts

```diff
@@ -3,3 +3,17 @@ export const getFakeEmail = () => {
   const array = new Uint32Array(1)
   return `${crypto.getRandomValues(array).toString()}@example.com`
 }
+
+export const allocateRaffle = async (): Promise<{ raffleId: number }> => {
+  const { metadata, hash } = await getImageInfo(selectedImage.value)
+  const body = {
+    email: getFakeEmail(),
+    hash,
+    address: accountId.value,
+    image: selectedImage.value,
+    metadata: metadata,
+  }
+
+  const response = await allocateCollection(body, drop.value.id)
+  return { raffleId: parseInt(response.result.id) }
+}
```

Comment:
Unfinished? Can't find name `getImageInfo, selectedImage, accountId, selectedImage, allocateCollection, drop`. Could you revert it instead and add it to https://github.com/kodadot/nft-gallery/issues/9635?

Reply:
hey

you maybe looked before i synced with main branch
this file is untouched by this PR

![image](https://github.com/kodadot/nft-gallery/assets/22791238/846ddcb4-ae1c-477c-8f8c-42172137527e)